util/p9/slist

```diff
@@ -0,0 +1,3 @@
+#! /bin/sh
+ls -all gatling-puppet-load-test/results/scale
```

Comment:
Should this have a target destination path set where the `gatling-puppet-load-test` directory is expected to be? Otherwise this command only works if you are in the correct `$(pwd)`.